client/elements/page-selector.html

```diff
@@ -229,6 +251,19 @@
               }
           }
 
+          ready() {
+              super.ready();
+              this.addEventListener('par-menu-copied', (e) => {
+                  this.toastMessage = e.detail.message;
+                  const success = e.detail.success;
+                  if (success) {
+                      this.$.success_toast.open();
+                  } else {
+                      this.$.error_toast.open();
+                  }
+              });
+          }
```

Comment:
Toasts are now displayed by sending a `show-sc-toast` event to the `sc-toasts.html` file. If you use that, you won't need to define any custom styles here. Check the `sc-donate-now-page` file for an example.

Reply:
I actually did not change anything to the code here: I just moved the code that was there from the sc-view-suttaplex to page-selector. That's all. The reason is that the toast for suttaplex cards was unable to show for pages other than in the sc-view-suttaplex like for instance the text pages. By moving it to higher up in the chain it could show up for any page.
Sure I can change the entire code, but considering that it is not my code in the first place I feel a little reluctant to do that.

Reply:
Oh, I see. That means we're still using the old style somewhere (that's an overshight, sorry).